nixos/platform/default.nix

```diff
@@ -48,6 +48,14 @@ in {
 
   options = with lib.types; {
 
+    flyingcircus.nix.connectTimeout = mkOption {
```

Comment:
Inventing a new option here that won't exist in any future release will break the unattended update in maintenance path, as the configuration will then only be valid for one of the platform versions.
Why not explicitly backport the option name `nix.settings.connect-timeout`, that will then be handled by generator-backed settings in future NixOS versions?